src/entities/Praise.ts

```diff
@@ -0,0 +1,59 @@
+import mongoose from 'mongoose';
+import { AccountSchema, UserAccountInterface } from './User';
+
+export interface QuantificationInterface {
+  createdAt: string;
+  updatedAt: string;
+  quantifierId: string;
+  score: number;
+  dismissed: boolean;
+  duplicatePraiseId: string;
+}
+
+export interface PraiseInputInterface {
+  periodId: number;
+  reason: string;
+  sourceId: string;
+  sourceName: string;
+  quantifications: Array<QuantificationInterface>;
+  giver: UserAccountInterface;
+  receiver: UserAccountInterface;
+}
+
+export interface PraiseDocument
+  extends PraiseInputInterface,
+    mongoose.Document {
+  createdAt: Date;
+  updatedAt: Date;
+}
+
+const QuantificationSchema = new mongoose.Schema(
+  {
+    id: { type: String, required: true, unique: true },
+    quantifierId: { type: String, required: true },
+    score: { type: Number, required: true },
```

Comment:
Score is not required. When we assign quantifiers we create quantification objects without scores.